x-pack/plugins/index_management/public/plugin.ts

```diff
@@ -6,54 +6,40 @@
 import { i18n } from '@kbn/i18n';
 
 import { CoreSetup } from '../../../../src/core/public';
+import { setExtensionsService } from './application/store/selectors/extension_service';
 
-import { UIM_APP_NAME, PLUGIN } from '../common/constants';
-
-import { httpService } from './application/services/http';
-import { notificationService } from './application/services/notification';
-import { UiMetricService } from './application/services/ui_metric';
-
-import { setExtensionsService } from './application/store/selectors';
-import { setUiMetricService } from './application/services/api';
+import { ExtensionsService } from './services';
 
 import { IndexManagementPluginSetup, SetupDependencies, StartDependencies } from './types';
-import { ExtensionsService } from './services';
 
 export class IndexMgmtUIPlugin {
-  private uiMetricService = new UiMetricService(UIM_APP_NAME);
   private extensionsService = new ExtensionsService();
 
   constructor() {
     // Temporary hack to provide the service instances in module files in order to avoid a big refactor
     // For the selectors we should expose them through app dependencies and read them from there on each container component.
     setExtensionsService(this.extensionsService);
-    setUiMetricService(this.uiMetricService);
   }
 
   public setup(
     coreSetup: CoreSetup<StartDependencies>,
     plugins: SetupDependencies
   ): IndexManagementPluginSetup {
-    const { http, notifications } = coreSetup;
     const { fleet, usageCollection, management } = plugins;
 
-    httpService.setup(http);
-    notificationService.setup(notifications);
-    this.uiMetricService.setup(usageCollection);
-
     management.sections.section.data.registerApp({
-      id: PLUGIN.id,
+      id: 'index_management',
```

Comment:
Is this considered better practice in general?

Reply:
ohh this i will revert. This shouldn't change anything. i was testing.